### PR TITLE
Add installation documentation for manual deployments.

### PR DESCRIPTION
**Pull request by gijskant**
Adding a new [installation manual](https://github.com/thehyve/glowing-bear/blob/installation-documentation/docs/installation.md) for manual deployments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
-# Glowing Bear
+# ![](docs/images/glowingbear.png) Glowing Bear
 [![Build Status](https://travis-ci.org/thehyve/glowing-bear.svg?branch=dev)](https://travis-ci.org/thehyve/glowing-bear/branches)
 [![codecov](https://codecov.io/gh/thehyve/glowing-bear/branch/dev/graph/badge.svg)](https://codecov.io/gh/thehyve/glowing-bear)
 
 An [Angular]-based frontend application for clinical data selection and analysis 
 based on [TranSMART]. Visit https://glowingbear.app for more information.
 
+## Usage
```
Comment (ewelinajozwik) [inline README.md]: Maybe without this header, just directly after the short information about gb? 
The image only doesn't say much about usage.

File: docs/installation.md [hunk 2]
```diff
@@ -0,0 +1,638 @@
...
+
+TranSMART API server requires:
+- JDK 8
+- a PostgreSQL database server
+
+Installation steps for TranSMART API server:
+
+1. Install the `pg_bitcount` extension, e.g., for PostgreSQL 11:
+   ```shell
+   POSTGRESQL_VERSION=12
```
Comment (ewelinajozwik) [inline docs/installation.md:222]: Not a problem, but seems a bit inconsistent that line above it mentions `e.g., for PostgreSQL 11` and here is `12`

File: docs/installation.md [hunk 3]
```diff
@@ -0,0 +1,638 @@
...
+
+5. Create a start script `/home/packer/start` with the configuration:
+   ```bash
+   #!/usr/bin/env bash
+   set -e
+   here=$(dirname "${0}")
+   source ${here}/venv/bin/activate
+
+   export KEYCLOAK_SERVER_URL=https://keycloak.example.com  # CHANGE ME
+   export KEYCLOAK_REALM=transmart  # CHANGE ME
```
Comment (ewelinajozwik) [inline docs/installation.md]: or 
```suggestion
   export KEYCLOAK_REALM=example  # CHANGE ME
```
to be consistent with configs specified for other applications.

File: docs/installation.md [hunk 4]
```diff
@@ -0,0 +1,638 @@
...
+   Please ensure that proper SSL certificates are installed.
+
+5. Reload nginx configuration:
+   ```shell
+   systemctl reload nginx.service
+   ```
+
+#### Availability check for monitoring
+
+Availability of the Keycloak application can be tested using the following public endpoints:
```
Comment (ewelinajozwik) [inline docs/installation.md]:
```suggestion
Availability of the Transmart API and GB backend applications can be tested using the following public endpoints:
```

**Review by ewelinajozwik — APPROVED** [review]
Great job with the documentation! It is very clear and descriptive.
I added few small comments, but in general looks great.

File: README.md (hunk 1 above)
Comment (gijskant) [reply README.md]: Good point. I think I intended to add some more information here, but I guess the 'Visit https://glowingbear.app for more information' covers that.

**Pull request merged**